.coafile

```diff
@@ -31,6 +31,10 @@ bears = SpaceConsistencyBear
 bears = GitCommitBear
 shortlog_trailing_period = False
 shortlog_regex = ([^:]*|[^:]+: [A-Z0-9*].*)
+body_close_issue = True
+body_close_issue_full_url = True
+body_close_issue_last_line = True
+body_enforce_issue_reference = False
```

Comment:
According to :point_up: [January 26, 2017 5:25 PM](https://gitter.im/coala/coala?at=5889cea3074f7be7631e1e30) , I believe we want this to be `True`.  Please correct me if I have misunderstood.
cc @Makman2.